susemanager-utils/testing/docker/scripts/init-pgsql-reportdb4eclipse.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -e
+
+cd /manager/susemanager-utils/testing/docker/scripts/
+
+# Move Postgres database to tmpfs to speed initialization and testing up
+if [ ! -z $PG_TMPFS_DIR ]; then
+    trap "umount $PG_TMPFS_DIR" EXIT INT TERM
+    ./docker-testing-pgsql-move-data-to-tmpfs.sh $PG_TMPFS_DIR
+fi
+
+export PERLLIB=/manager/spacewalk/setup/lib/:/manager/web/modules/rhn/:/manager/web/modules/pxt/:/manager/schema/spacewalk/lib
+export PATH=/manager/schema/spacewalk/:/manager/spacewalk/setup/bin/:$PATH
+
+echo Going to reset pgsql database
+
+echo $PATH
+echo $PERLLIB
+
+export SYSTEMD_NO_WRAP=1
```

Comment:
I would like to suggest relocating all the `export` statements to the beginning of the file and refraining from using `echo` directly.

Reply:
Not in this PR. Feel free to start an own

Reply:
ok